tests/layer_tests/tensorflow_tests/test_tf_Sub.py

```diff
@@ -8,14 +8,16 @@
 
```

Comment:
please revert your changes for `test_tf_Sub.py` and add separate test suite for complex type